webrtc-stats.html

```diff
@@ -924,7 +926,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the number of unique datachannels opened.
+                  Represents the number of unique DataChannels returned from a successful
+                  createDataChannel() call.
```

Comment:
Tried again - this time by defining "opened" as "has entered the open state" and "closed" as "has left the open state", and a new counter "dataChannelsRequested".